src/components/project/project.repository.ts

```diff
@@ -115,6 +115,11 @@ export class ProjectRepository extends CommonRepository {
           relation('out', '', 'owningOrganization', ACTIVE),
           node('organization', 'Organization'),
         ])
+        .optionalMatch([
+          node('node'),
+          relation('out', '', '', ACTIVE),
+          node('marketingRegionOverried', 'Location'),
+        ])
```

Comment:
@atGit2021 This is the part I am not to sure about.  See what you think?

Reply:
I think it may be missing the relation name in the third param, line 120.